cnxarchive/views/content.py

```diff
@@ -241,6 +244,8 @@ def get_content_html(request):
     resp.status = "200 OK"
     resp.content_type = 'application/xhtml+xml'
     resp.body = content
+    if result['baked'] is not None:
+        resp.cache_control = "no-cache"
```

Comment:
Ditto. Also, on deeper look at the pyramid `response` object API, let's use `resp.cache_expires(0)` instead of setting the header directly. This will have nice affects for using the http_cache machinery later to set values for caching via config. https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/webob.html?highlight=response.cache_expires